.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,7 +13,7 @@ assignees: ''
 - `echo $LS_COLORS`: 
 
 ## Expected behavior
-If applicable, add the output of the classic ls command (`\ls -la`) in order to show the buggy file/directory.
```

Comment:
Hello!

The '\' is used to ensure that we use the real ls and not some aliases